### DISCUSSION TITLE
Allow Theming of Settings

### DISCUSSION THREAD

**Discussion by PsycloneTM** [Chatterino/chatterino2#4679] [Ideas] [answered]
Currently on all themes the settings page loads with the default theme of dark. I believe we should be able to customize it to our liking.

**Comment by pajlada** [comment]
If we do, there's a possibility of a user accidentally loading a theme that make the settings dialog unusable

I also wonder how worth it is when the settings page is very rarely opened by users - the amount of extra complexity in implementation & maintenance feels like a burden for the small gain

**Comment by 8thony** [comment, marked as answer]
- Duplicate of #4823
